### DISCUSSION TITLE
Statistics in the about page

### DISCUSSION THREAD

**Discussion by Nikos1508** [simplified-coding/simplified-coding#137] [Ιδέες / Σχόλια]
I think that we should add some statistics (mainly from our github) in our about page. We cam also animate the numbers so it looks better and cooler

**Comment by odysseuskir** [comment]
That's a cool idea. I also propose we put a table with the current roadmap that we are working on.

**Comment by Nikos1508** [comment]
YES

**Reply by odysseuskir** [reply]
Please reply to a comment, DON'T make a new comment and don't make multiple, as it spams our emails.

**Comment by Nikos1508** [comment]
But these are things that we could write in the newsletter.
